mne/viz/utils.py

```diff
@@ -897,3 +901,126 @@ def _process_times(inst, times, n_peaks=None, few=False):
                            'than 20 time instants.')
 
     return times
+
+
+def plot_sensors(info, kind='topomap', ch_type=None, title=None,
+                 show_names=False, show=True):
+    """Plot sensors positions.
+
+    Parameters
+    ----------
+    info : Instance of Info
+        Info structure containing the channel locations.
+    kind : str
+        Whether to plot the sensors as 3d or as topomap. Available options
+        'topomap', '3d'. Defaults to 'topomap'.
+    ch_type : 'mag' | 'grad' | 'eeg' | 'seeg' | None
+        The channel type to plot. If None, then channels are chosen in the
+        order given above.
+    title : str | None
+        Title for the figure. If None (default), equals to
+        ``'Sensor positions (%s)' % ch_type``.
+    show_names : bool
+        Whether to display all channel names. Defaults to False.
+    show : bool
+        Show figure if True. Defaults to True.
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        Figure containing the sensor topography.
+
+    Notes
+    -----
+    This function plots the sensor locations from the info structure using
+    matplotlib. For drawing the sensors using mayavi see
+    :func:`mne.viz.plot_trans`.
+    See also :func:`mne.viz.plot_layout`.
+
+    .. versionadded:: 0.12.0
+
+    """
+    if kind not in ['topomap', '3d']:
+        raise ValueError("Kind must be 'topomap' or '3d'.")
+    ch_indices = channel_indices_by_type(info)
+    if ch_type is None:
+        for type in ['mag', 'grad', 'eeg', 'seeg']:
+            if _contains_ch_type(info, type):
+                ch_type = type
+                break
+    elif ch_type not in ['mag', 'grad', 'eeg', 'seeg']:
+        raise ValueError("ch_type must be one of ['mag', 'grad', 'eeg', "
+                         "'seeg'] not %s!" % ch_type)
```

Comment:
maybe a local variable with supported types to avoid repetition and facilitate updates in the future